tests/reals/real_clds_singly_linked_list.h

```diff
@@ -3,13 +3,13 @@
 #ifndef REAL_CLDS_SINGLY_LINKED_LIST_H
 #define REAL_CLDS_SINGLY_LINKED_LIST_H
 
-#include "macro_utils.h"
+#include "azure_macro_utils/macro_utils.h"
 #include "clds/clds_singly_linked_list.h"
 
 #define R2(X) REGISTER_GLOBAL_MOCK_HOOK(X, real_##X);
 
 #define REGISTER_CLDS_SINGLY_LINKED_LIST_GLOBAL_MOCK_HOOKS() \
-    FOR_EACH_1(R2, \
```

Comment:
>FOR_EACH_1 [](start = 4, length = 10)

interesting. How did this macro work before?

Reply:
Also, if macro_utls  is a local copy, is it going to be deleted?

---
In reply to: [370396422](https://github.com/dcristoloveanu/clds/pull/39#discussion_r370396422) [](ancestors = 370396422)

Reply:
Well, this real was not used. And at some point there was a local macro_utils.h I believe.